qmk_firmware/keyboards/keyball/lib/keyball/keyball.c

```diff
@@ -500,8 +500,8 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
         // process KC_MS_BTN1~8 by myself
         // See process_action() in quantum/action.c for details.
         case KC_MS_BTN1 ... KC_MS_BTN8: {
-            extern void register_button(bool, enum mouse_buttons);
-            register_button(record->event.pressed, MOUSE_BTN_MASK(keycode - KC_MS_BTN1));
+            extern void register_mouse(uint8_t mouse_keycode, bool pressed);
+            register_mouse(keycode, record->event.pressed);
```

Comment:
I believe this one is not needed anymore.
https://github.com/qmk/qmk_firmware/commit/dbe214c3268d9a316a137471a7679f7e45046e49

If I'm not wrong, then the process_action() already handles this special case.